tests/ts_replication.erl

```diff
@@ -104,6 +106,22 @@ replication(ANodes, BNodes, Table, NormalType) ->
     create_normal_type(ANodes, NormalType),
     create_normal_type(BNodes, NormalType),
 
+    %% Make certain we don't start throwing objects into the realtime
```

Comment:
Should this same intercept/test be done in other places (fullsync test, for example, to make sure we're not pushing data if fullsync is `both` or `fullsync`?